src/core/dapps/build_in/utxo.cr

```diff
@@ -65,6 +65,7 @@ module ::Sushi::Core::DApps::BuildIn
       pay_token = sender[:amount]
       pay_default = (transaction.token == DEFAULT ? sender[:amount] : 0_i64) + sender[:fee]
 
+      # TODO: Fix the error wording here. Needs discussion.
```

Comment:
This message shows when the user is trying to send more tokens than he has available in his wallet and there are 2 scenarios:

  1. custom token 
  2. default SUSHI token

so I think the wording should be something like:

Unable to send <amount_to_send> <token> to recipient because you do not have enough <token>

Reply:
Beautiful